config/dispatcher/dispatcher.ex

```diff
@@ -111,6 +111,10 @@ defmodule Dispatcher do
     Proxy.forward conn, path, "http://publisher/regulatory-attachment-publication-tasks/"
   end
 
+  match "/snippet-lists/*path", %{ accept: %{json: true}, layer: :api} do
+    Proxy.forward conn, path, "http://resource/snippet-lists/"
+  end
+
```

Comment:
any reason we don't want to cache this? As far as I know I don't think it would give a problem, and if these get used a lot, caching will be useful.